pkg/loader/loader.go

```diff
@@ -0,0 +1,154 @@
+// SPDX-FileCopyrightText: 2022-present Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package loader
+
+import (
+	"context"
+	"fmt"
+	simapi "github.com/onosproject/onos-api/go/onos/fabricsim"
+	"github.com/onosproject/onos-lib-go/pkg/logging"
+	"google.golang.org/grpc"
+)
+
+var log = logging.GetLogger()
+
+// LoadTopology loads the specified YAML file and creates the prescribed simulated topology entities
+// using the fabric simulator API client.
+func LoadTopology(conn *grpc.ClientConn, topologyPath string) error {
+	topology := &Topology{}
+	if err := LoadTopologyFile(topologyPath, topology); err != nil {
+		return err
+	}
+
+	log.Infof("%+v", topology)
```

Comment:
Perhaps make this debug. I assume it is gonna be a big log

Reply:
Yes, I'll remove this... it's here purely for initial signal of activity. :)